SCClassLibrary/Common/Core/Boolean.sc

```diff
@@ -18,9 +18,13 @@ Boolean {
 	keywordWarnings {
 		// turn on/off warnings if a keyword argument is not found
 		_KeywordError
+		^this.primitiveFailed
+	}
+	trace {
+		// this is only available in a special debugging version of the app
```

Comment:
:) 

https://medium.freecodecamp.com/the-100-correct-coding-style-guide-5b594a1655f0